WalletWasabi/Models/PreferredScriptPubKeyType.cs

```diff
@@ -0,0 +1,31 @@
+using NBitcoin;
+using WalletWasabi.Extensions;
+
+namespace WalletWasabi.Models;
+
+public abstract class PreferredScriptPubKeyType
+{
+	private PreferredScriptPubKeyType() { }
+
+	public sealed class Unspecified : PreferredScriptPubKeyType
+	{
+		public static readonly Unspecified Instance = new ();
+
+		public string Name { get; } = "Random";
```

Comment:
I'd call this one `SegWit & Taproot`
and the others `SegWit only` & `Taproot Only`

Reply:
done